hardware_enablement/kmm-release-notes.adoc

```diff
@@ -297,7 +297,7 @@ For more information, see xref:../hardware_enablement/kmm-kernel-module-manageme
 
 === Known issues
 // MGMT-20453 changed security settings in jira
-* The `ModuleUnloaded` event does not appear when a module is `Unloaded``.
+* The `ModuleUnloaded` event does not appear when a module is `Unloaded`.
```

Comment:
Probably our of scope, but a consideration:

> The `ModuleUnloaded` event does not show when a module is `Unloaded`.


<img width="725" height="122" alt="Screenshot From 2025-07-11 14-44-08" src="https://github.com/user-attachments/assets/842413ed-8504-40e9-bb53-e24dbee99b5e" />

Reply:
It's a bit more complicated than that. Also, has already been published.